src/smart-components/role/add-role/schema.js

```diff
@@ -13,11 +13,13 @@ import messages from '../../../Messages';
 import providerMessages from '../../../locales/data.json';
 import { validateNextAddRolePermissionStep } from '../permission-wizard-helper';
 import InventoryGroupsRoleTemplate from './inventory-groups-role-template';
-import { enableWorkspacesNameChange } from '../../../helpers/role/inventory-helper';
+import { useFlag } from '@unleash/proxy-client-react';
 
 export const schemaBuilder = (container) => {
   const cache = createIntlCache();
   const intl = createIntl({ locale, messages: providerMessages }, cache);
+  const enableWorkspacesNameChange = useFlag('platform.rbac.groups-to-workspaces-rename');
```

Comment:
The hook should not be used here. Please, let's pass the flag value as a prop instead.